aten/src/ATen/native/quantized/cpu/qclamp.cpp

```diff
@@ -41,6 +41,42 @@ Tensor quantized_clamp(
   return qy;
 }
 
+// hardtanh is clamp with default min==-1.0f and default max==1.0f
+Tensor quantized_hardtanh(
+    const Tensor& qx,
+    Scalar min,
+    Scalar max) {
+  Tensor qy;
+  AT_DISPATCH_QINT_TYPES(qx.scalar_type(), "hardtanh", [&]() {
+    qy = quantized_clamp_impl(qx, min, max);
+  });
```

Comment:
I don't think we need to dispatch here:

```python
qy = quantized_clamp_impl(qx, min, max);
```

should do the trick.

Reply:
> I don't think we need to dispatch here:

sounds good.  Just for my own understanding, the reason we don't need the dispatch is because there is no need to branch things by `scalar_t`?  Originally I added the dispatch as a copy-pasta from clamp above.

Reply:
Yea, I think the dispatch was a contamination from earlier versions of the code. But it is still worth testing if we really need it